src/components/SingleReview.jsx

```diff
@@ -44,17 +69,40 @@ export function SingleReview() {
       <b>{review.category}</b>
       <p>Created by {review.designer}</p>
       <h3>{review.owner}</h3>
+      <Avatar avatarUrl={avatarUrl} />
       <p>{review.created_at}</p>
       <article>
         <p>{review.review_body}</p>
       </article>
-      <p>Votes: {review.votes}</p>
+      <p>Votes: {reviewVoteCount}</p>
+      <VoteButton
+        id={review.review_id}
+        type='review'
+        setVoteCount={setReviewVoteCount}
+        setDisplayAlert={setDisplayAlert}
+      />
+      {displayAlert && <Alert severity='error' crud='Vote' setDisplayAlert={setDisplayAlert} />}
+      {isLoggedOut(userLogin) ? (
+        <p><b>Login to post a comment</b></p>
+      ) : (
+        <CommentForm setComments={setComments} reviewID={review.review_id} setReview={setReview} />
+      )}
       <p>Comments: {review.comment_count}</p>
-      <>{commentsIsLoading ? <p>'Loading...'</p> : <ul>{comments.map((comment) => {
-        return <li key={comment.comment_id}>
-            <CommentCard comment={comment}/>
-        </li>
-      })}</ul>}</>
+      <>
+        {commentsIsLoading ? (
+          <p>'Loading...'</p>
+        ) : (
+          <ul>
```

Comment:
You can remove the bullet points beside the comment boxes to make it look cleaner. I'd look into adding **list-style: none** in your CSS file 👍 That way, the box should be aligned with the rest of the text on the page 🤸